include/openssl/self_test.h

```diff
@@ -68,6 +68,7 @@ extern "C" {
 # define OSSL_SELF_TEST_DESC_KDF_SSHKDF     "SSHKDF"
 # define OSSL_SELF_TEST_DESC_KDF_TLS12_PRF  "TLS12_PRF"
 # define OSSL_SELF_TEST_DESC_KDF_KBKDF      "KBKDF"
+# define OSSL_SELF_TEST_DESC_KDF_TLS13_KDF  "TLS13_KDF"
```

Comment:
doc/man7/OSSL_PROVIDER-FIPS.pod also needs updating.

I would suggest using 2 names here "TLS13_KDF_EXTRACT" and ""TLS13_KDF_EXPAND" maybe? And then just using those names in the self test.

Reply:
Or PSK??

Reply:
I went with extract and expand.  Otherwise, I'd need a second meaning ful name.